src/gms/project/README.md

```diff
@@ -0,0 +1,37 @@
+# Docs
+
+## How to use
+1. Import the package (`Demo` folder is optional).
+2. Create `Sessionless` context object.
+3. Remember to call `Clean()` method whenever the context object is destroyed.
+
+## `Sessionless`
+Sessionless context object is an object that lets you communicate with the underlying engine.
+In most cases you don't have to create it more than once in the entire lifetime of the process.
+It's allocated globally on the heap, and it is your responsibility to deallocate it whenever it's no longer used.
+
+The context object can be created as follows:
+```js
```

Comment:
is this the right syntax type?

Reply:
There is no support for GML, but the JS syntax have the best coloring scheme